features/features.go

```diff
@@ -393,6 +393,14 @@ var (
 					enableIn(configv1.Default, configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
 					mustRegister()
 
+	FeatureGateManagedBootImagesvSphere = newFeatureGate("ManagedBootImagesvSphere").
+					reportProblemsToJiraComponent("MachineConfigOperator").
+					contactPerson("rsaini").
+					productScope(ocpSpecific).
+					enhancementPR("https://github.com/openshift/enhancements/pull/1496").
+					enableIn(configv1.Default, configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
```

Comment:
You can't add the gate to the default feature set from the get go

Reply:
apologies. Updated